internal/rest/dequeue_nodes.go

```diff
@@ -399,21 +399,23 @@ func (rest *RestOperations) ExecuteRestAndPopulateCache(rest_ops []*utils.RestOp
 									utils.AviLog.Infof("key: %s, msg: Error is not of type AviError, err: %v, %T", key, rest_ops[i].Err, rest_ops[i].Err)
 									continue
 								}
-								retryable, fastRetryable := rest.RefreshCacheForRetryLayer(publishKey, aviObjKey, rest_ops[i], aviError, aviclient, avimodel, key)
-								fastRetry = fastRetry || fastRetryable
-								retry = retry || retryable
+								regularRetry, slowRetry = rest.RefreshCacheForRetryLayer(publishKey, aviObjKey, rest_ops[i], aviError, aviclient, avimodel, key)
```

Comment:
RefreshCacheForRetryLayer would return true always for regular retry, so we would never go to slow retry.

Reply:
Retained the logic that was there earlier and handling the slow retry checks separately.